package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "space-hub",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.6",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.1.2",
+    "i": "^0.3.7",
+    "npm": "^8.19.2",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.4",
+    "react-router-dom": "^6.4.2",
+    "react-scripts": "5.0.1",
```

Comment:
- [ ] Great job installing/using react-router and react-redux, however it's one of the requirement to also install redux logger kindly integrate that to your project